nav2_msgs/msg/TrackingError.msg

```diff
@@ -0,0 +1,8 @@
+# Real-time tracking error for Nav2 controller
+
+std_msgs/Header header
+float32 tracking_error
```

Comment:
Add a note about the positive vs negative for the left/right deviation from the path

Reply:
@silanus23 please do this :-)